.idea/codeStyleSettings.xml

```diff
@@ -102,5 +104,4 @@
     </option>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
   </component>
-</project>
-
+</project>
```

Comment:
The file is managed by IntelliJ and it does not end the file with EOL. I suggest we keep as is.